pkg/types/kubetoken.go

```diff
@@ -0,0 +1,18 @@
+package types
+
+import (
+	"encoding/json"
+	authenticationv1 "k8s.io/api/authentication/v1"
+)
+
+type KubeTokenResponse struct {
+	authenticationv1.TokenRequest
+}
+
+func (k *KubeTokenResponse) ToJson() (string, error) {
```

Comment:
NIT: types package should not contain any method. ToJson should be a struct that transform a kubetoken to json. If we want to extend functionality and create a `ToYAML`  we don't have to modify `KubeTokenResponse` WDYT?

Reply:
makes sense, done in 1945273c3da13950ac31a1fe3162716b6e580d2e